product_docs/docs/tde/15/enabling_tde_pg_upgrade.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Enabling TDE with pg_upgrade"
+navTitle: Enabling TDE with pg_upgrade
+---
+
+EDB supports using [pg_upgrade](/tde/latest/pg_upgrade/) to add encryption to unencrypted systems. 
+This utility allows you to enable TDE when performing an upgrade to a more recent version, when transitioning from an open-source 
+postgreSQL instance to an EDB-supported instance, or to enable encryption while maintaining your current instance.
+
+You can also use `pg_upgrade` on already encrypted systems to rotate the data encryption keys. 
+
+Here are some example use cases:
+
+| Source system                               | Target system                                                      |
+|---------------------------------------------|--------------------------------------------------------------------|
+| Unencrypted EDB Postgres Extended Server 16 | Encrypted EDB Postgres Extended Server 16                          |
+| Unencrypted PostgreSQL 15                   | Encrypted EDB Postgres Extended Server 16                          |
+| Unencrypted PostgreSQL 16                   | Encrypted EDB Postgres Extended Server 16                          |
+| Encrypted EDB Postgres Extended Server 15   | Encrypted EDB Postgres Extended Server 16 with new encryption keys |
+
+## Overview 
+
+1. Perform a backup of your system. 
+1. Install the target system version, for example, EDB Postgres Extended Sever, or EDB Postgres Advanced Server. 
+1. Initialize a target instance and enable TDE encryption. 
+1. Use `pg_upgrade` with the `--copy-by-block` option to upgrade to a TDE system.
+
+## Example 
+
+This section displays using `pg_upgrade` to transition from an unencrypted PostgreSQL 16 system to an encrypted 
+EDB PostgreSQL Extender Server 16 on a Linux platform.
+
+### Step 1 - Perform a backup of your source system 
+
+Perform a backup of your system. You can use [Barman](https://www.enterprisedb.com/docs/supported-open-source/barman/), 
+[pg_dumpall](https://www.postgresql.org/docs/current/app-pg-dumpall.html) or a different backup utility.
+
+### Step 2 - Install the required target system
+
+Install the product where you want to create the target system, in this example, 
+[EDB Postgres Extended Server on Linux](/pge/latest/installing/).
+
+### Step 3 - Initialize the target instance
+
+Initialize a cluster in the target instance. The cluster must:
+
+  * Use wrap and unwrap data encryption keys.
+  * Support data encryption with the `--data-encryption` option. 
+  * Be accessible through a different port than the source instance.
+
+See [Using initdb TDE options](/tde/latest/enabling_tde/#using-initdb-tde-options) and this 
+[example](/tde/latest/enabling_tde/#example) for more information.
+
+[not sure if we need to start a database server as well.]: #
```

Comment:
Do we need a server to have started at this point? In my test, I did, but maybe not a required step.